libtest/include.am

```diff
@@ -192,6 +193,7 @@ $(TMP_DIR):
 	@$(mkdir_p) $(TMP_DIR)
 
 libtest_unittest_LDADD+= libtest/libtest.la
+libtest_unittest_LDADD+= $(libmemcached_libmemcached_la_OBJECTS)
 libtest_unittest_SOURCES= libtest/unittest.cc
```

Comment:
Modified the automake include files to fix compiling/linking errors when referencing memcached functions